flink-streaming-connectors/flink-connector-cassandra/pom.xml

```diff
@@ -49,7 +50,9 @@ under the License.
 				<configuration>
 					<reuseForks>true</reuseForks>
 					<forkCount>1</forkCount>
-					<argLine>-Xms256m -Xmx2800m -Dmvn.forkNumber=${surefire.forkNumber} -XX:-UseGCOverheadLimit</argLine>
+					<systemPropertyVariables>
+						<mvn.forkNumber>0${surefire.forkNumber}</mvn.forkNumber>
+					</systemPropertyVariables>
```

Comment:
We can just remove all argLine definition here along with the forkNumber definition. They are already defined in the parent pom.